jaguarbot:
  # Controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  left_joint_1_controller:
    type: effort_controllers/JointVelocityController
    joint: left_joint_1
    pid: {p: 1, i: 0, d: 0}

  left_joint_2_controller:
    type: effort_controllers/JointVelocityController
    joint: left_joint_2
    pid: {p: 1, i: 0, d: 0}

  left_joint_3_controller:
    type: effort_controllers/JointVelocityController
    joint: left_joint_3
    pid: {p: 1, i: 0, d: 0}

  right_joint_1_controller:
    type: effort_controllers/JointVelocityController
    joint: right_joint_1
    pid: {p: 1, i: 0, d: 0}

  right_joint_2_controller:
    type: effort_controllers/JointVelocityController
    joint: right_joint_2
    pid: {p: 1, i: 0, d: 0}

  right_joint_3_controller:
    type: effort_controllers/JointVelocityController
    joint: right_joint_3
    pid: {p: 1, i: 0, d: 0}
